packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -364,9 +364,8 @@ model = GLiNER.from_pretrained("${model.id}")`,
 ];
 
 export const keras = (model: ModelData): string[] => [
-	`# Available backend options are: "jax", "tensorflow", "torch".
-import os
-os.environ["KERAS_BACKEND"] = "tensorflow"
+	`import os
+os.environ["KERAS_BACKEND"] = "jax" # or "torch", or "tensorflow"
```

Comment:
let's just change the deafault to jax but not reformat?

Reply:
sure